testing/testdata/script/create-project.txtar

```diff
@@ -0,0 +1,20 @@
+# Create a project successfully without the github flag:
+env PROJECT_NAME=test-project-$RESIM_TEST_UUID
+exec echo $PROJECT_NAME
+exec resim project create --name $PROJECT_NAME --description "This is a test project"
+! stderr . # no errors
+stdout 'Creating a project...'
```

Comment:
This is a little brittle, I would rather maybe just check the exit code if we can?

Reply:
Fair